src/Compilers/CSharp/Portable/CSharpCodeAnalysis.csproj

```diff
@@ -892,17 +893,16 @@
     </BoundTreeDefinition>
     <Compile Include="$(IntermediateOutputPath)BoundNodes.xml.Generated.cs" />
     <Compile Include="$(IntermediateOutputPath)ErrorFacts.Generated.cs" />
-    <Compile Include="$(IntermediateOutputPath)Syntax.xml.Generated.cs" />
+    <Compile Include="$(IntermediateOutputPath)Syntax.xml.Syntax.Generated.cs" />
```

Comment:
You could try putting it in the target, but you then need to ensure that runs during design time builds. I guess this is fine then.